tensorflow/core/grappler/optimizers/remapper.cc

```diff
@@ -649,6 +650,34 @@ bool FindContractionWithBias(const RemapperContext& ctx, int node_index,
   return true;
 }
 
+bool RuntimeFusionEnabled(const Cluster& cluster) {
+  static bool is_enabled = [&] {
```

Comment:
This being `static` but depending on `cluster` seems unsafe? Is it possible for `cluster` to change in different calls?
Also it should probably be `const`.

Reply:
That's a good cache. But I feel it should be fine in this case. It looks like the `cluster` is a collection of hardware resources and I don't think it will change during the TF model execution.